frontends/mit-open/src/page-components/Header/UserMenu.tsx

```diff
@@ -1,61 +1,103 @@
 import React, { useState } from "react"
-import { Avatar, Badge, SimpleMenu, styled } from "ol-components"
-import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown"
-import ArrowDropUpIcon from "@mui/icons-material/ArrowDropUp"
-import type { BadgeProps, SimpleMenuItem, AvatarProps } from "ol-components"
+import { ActionButtonLink, ButtonLink, SimpleMenu, styled } from "ol-components"
+import type { MenuOverrideProps, SimpleMenuItem } from "ol-components"
 import * as urls from "@/common/urls"
-import PersonIcon from "@mui/icons-material/Person"
-import { useLocation } from "react-router"
+import {
+  RiAccountCircleFill,
+  RiArrowUpSLine,
+  RiArrowDownSLine,
+} from "@remixicon/react"
 import { useUserMe, User } from "api/hooks/user"
+import { useLocation } from "react-router"
 
-const StyledBadge = styled(Badge)`
-  pointer-events: none;
-`
+const FlexContainer = styled.div({
+  display: "flex",
+  alignItems: "center",
+})
 
-const SmallAvatar = styled(Avatar)<AvatarProps>`
-  pointer-events: all;
-  height: 35px;
-  width: 35px;
-  border: none;
-  font-size: 1rem;
-  cursor: pointer;
-`
+const UserMenuContainer = styled.button({
+  display: "flex",
+  cursor: "pointer",
+  background: "none",
+  color: "inherit",
+  border: "none",
+  padding: "0",
+  font: "inherit",
+})
 
-const badgeAnchorOrigin: BadgeProps["anchorOrigin"] = {
-  vertical: "bottom",
-  horizontal: "right",
-}
+const LoginButtonContainer = styled(FlexContainer)(({ theme }) => ({
+  paddingRight: "32px",
+  "&:hover": {
+    textDecoration: "none",
+  },
+  [theme.breakpoints.down("sm")]: {
+    padding: "0",
+    ".login-button-desktop": {
+      display: "none",
+    },
+    ".login-button-mobile": {
+      display: "flex",
+    },
+  },
+  [theme.breakpoints.up("sm")]: {
+    ".login-button-desktop": {
+      display: "flex",
+    },
+    ".login-button-mobile": {
+      display: "none",
+    },
+  },
+}))
 
-type AuthMenuItem = SimpleMenuItem & {
+const UserIcon = styled(RiAccountCircleFill)(({ theme }) => ({
+  width: "20px",
```

Comment:
Width and height 24px.

Reply:
Ah, I see. I was confused by this:

<img width="419" alt="image" src="https://github.com/mitodl/mit-open/assets/12089658/6d75f555-6f57-4798-81fc-3ee20e95c947">

As you can see, it says the icon itself is 20x20, but is stretched to 24x24 in Figma.